### DISCUSSION TITLE
Make the scripts repository agnostic

### DISCUSSION THREAD

**Discussion by MacWeber** [tteck/Proxmox#622] [Ideas]
There are reasons to make the scrips run not depending on a particular git repository. For example:
- Debugging the code would be easier
- Adding new scripts would let them to be better tested locally

There could be the option to use a different repository, but another option could be running the scripts locally not depending on downloading other scripts from remotes. A use case for the second option would be in the scenario that the hypervisor or the VM has no access to GitHub.

**Comment by SimonMcN** [comment]
This is a good idea imo.  a git clone and local, or storing the repo in a variable would be nice.  I hate running bash scripts from remote locations without having to check them each time.
